### ISSUE TITLE
Difference between the files

### ISSUE THREAD

**Issue by MathiasHv18** [jdp111/HSK-JSON-files#1]
Hi! I found your json very usefull and I would like to use it, but I was wondering whats the difference between the HSK_lvlX_combed and the hsk_LX_online, since there is a big differences in the amount of lines.

**Comment by jdp111** [comment]
Yeah sorry the early stages of this project were a mess for me. The
"combed" files are the ones I got from a friend as an excel file, and it's
ordered by usage, so the most commonly used characters are first, which is
nice for first time learners.
The "online" files are scrubbed from
https://mandarinbean.com/new-hsk-1-word-list/ , ordered alphabetically by
pinyin.
they will share a lot of the same words, but I'm not sure if the online
file contains all the words from the combed files. When I implemented it, I
just set up my database to skip duplicates
The files were mainly for a personal project but of course you're welcome
to use it
